ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -758,9 +830,9 @@ impl<'a> Assemble<'a> {
                 VarLocation::Register(reg_alloc::Register::GP(Rq::RAX))
             }
             yksmp::Location::Register(1, ..) => {
-                // The third argument to the control point is the stackmap ID and is passed by RDX.
-                // We don't anticipate this ever being a live variable.
-                unreachable!()
+                // While we would never see this case in normal traces (since the control point
```

Comment:
I think we can just say "This case only happens in side-traces" or something equally simple.

Reply:
[6ca1b84](https://github.com/ykjit/yk/pull/1422/commits/6ca1b8481135aacac479f697b24350d98ff83886)